addons/web/static/src/views/fields/progress_bar/progress_bar_field.js

```diff
@@ -150,10 +156,13 @@ ProgressBarField.displayName = _lt("Progress Bar");
 ProgressBarField.supportedTypes = ["integer", "float"];
 
 ProgressBarField.extractProps = ({ attrs }) => {
+    if (!isNaN(attrs.options.max_value)) {
+        browser.console.warn(`${attrs.options.max_value} cannot be set as max_value. You must use the name of another field in the view.`);
+    }
```

Comment:
not sure it is necessary (nor the test)

Reply:
DBO suggested me to throw a message if a developer used wrongly the max_value attribute

Reply:
I added the test as a way to "write specs" regarding the non-support of numeric values for the option. I can remove it